### PR TITLE
add Contribution guide for Gradio Demo for ONNX organization

### PR DESCRIPTION
**Pull request by AK391**
added contribution guide for PRs adding Gradio demos to ONNX organization

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ model_proto = onnx.load("path to .onnx file")
 checker.check_model(model_proto)
 ```
 
+### Contribute a Gradio Demo to ONNX Organization on Hugging Face
+
+* Create an account on Hugging Face: https://huggingface.co/join
+* See list of models left to add to ONNX organization, please refer to the table with the [Models list](https://github.com/onnx/models)
```
Comment (jcwchen) [inline .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
* See list of models left to add to ONNX organization, please refer to the table with the [Models list](https://github.com/onnx/models#models)
```
nit: link to more precise section. L22 ditto.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -12,6 +12,15 @@ model_proto = onnx.load("path to .onnx file")
...
+* Add Gradio Demo under your username, see blog for setting up Gradio Demo on Hugging Face: https://huggingface.co/blog/gradio-spaces
+* request to join ONNX Organization: https://huggingface.co/onnx
+* once approved transfer model from your username to ONNX organization
+* add badge for model in model table, see examples in [Models list](https://github.com/onnx/models)
```
Comment (jcwchen) [inline .github/PULL_REQUEST_TEMPLATE.md:22]:
```suggestion
* Add a badge for model in model table, see examples in [Models list](https://github.com/onnx/models)
```
Capital case. L20 and L21 ditto.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (AK391) [reply .github/PULL_REQUEST_TEMPLATE.md]: @jcwchen thanks updated

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -12,6 +12,15 @@ model_proto = onnx.load("path to .onnx file")
 checker.check_model(model_proto)
 ```
 
+### Contribute a Gradio Demo to ONNX Organization on Hugging Face
+
+* Create an account on Hugging Face: https://huggingface.co/join
+* See list of models left to add to ONNX organization, please refer to the table with the [Models list](https://github.com/onnx/models#models)
+* Add Gradio Demo under your username, see blog for setting up Gradio Demo on Hugging Face: https://huggingface.co/blog/gradio-spaces
+* request to join ONNX Organization: https://huggingface.co/onnx
+* once approved transfer model from your username to ONNX organization
```
Comment (jcwchen) [inline .github/PULL_REQUEST_TEMPLATE.md:21]:
```suggestion
* Once approved transfer model from your username to ONNX organization
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -12,6 +12,15 @@ model_proto = onnx.load("path to .onnx file")
 checker.check_model(model_proto)
 ```
 
+### Contribute a Gradio Demo to ONNX Organization on Hugging Face
+
+* Create an account on Hugging Face: https://huggingface.co/join
+* See list of models left to add to ONNX organization, please refer to the table with the [Models list](https://github.com/onnx/models#models)
+* Add Gradio Demo under your username, see blog for setting up Gradio Demo on Hugging Face: https://huggingface.co/blog/gradio-spaces
+* request to join ONNX Organization: https://huggingface.co/onnx
```
Comment (jcwchen) [inline .github/PULL_REQUEST_TEMPLATE.md:20]:
```suggestion
* Request to join ONNX Organization: https://huggingface.co/onnx
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -12,6 +12,15 @@ model_proto = onnx.load("path to .onnx file")
...
+* once approved transfer model from your username to ONNX organization
+* Add a badge for model in model table, see examples in [Models list](https://github.com/onnx/models)
```
Comment (jcwchen) [inline .github/PULL_REQUEST_TEMPLATE.md:22]:
```suggestion
* Add a badge for model in model table, see examples in [Models list](https://github.com/onnx/models#models)
```

**Review by jcwchen — COMMENTED** [review]
Final nit comments. Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (AK391) [reply .github/PULL_REQUEST_TEMPLATE.md]: thanks updated

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (AK391) [reply .github/PULL_REQUEST_TEMPLATE.md]: updated

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (AK391) [reply .github/PULL_REQUEST_TEMPLATE.md]: updated

**Comment by liqunfu** [comment]
@AK391 
Thank you very much for contributing to ONNX model-zoo from last year!

We are updating tutorial/model [contributor list](https://github.com/onnx/sigs/blob/main/CONTRIBUTORS) with contributors from last year. I wonder if there is anyone that has made significant contribution to models in your PR, please let me know so I can add them to the list.

Thank you!

Liqun

**Pull request merged**